services/requester/keystore.go

```diff
@@ -0,0 +1,111 @@
+package requester
+
+import (
+	"fmt"
+	"sync"
+
+	flowsdk "github.com/onflow/flow-go-sdk"
+
+	"github.com/onflow/flow-go-sdk/crypto"
+)
+
+var ErrNoKeysAvailable = fmt.Errorf("no keys available")
```

Comment:
**Note:** Implementation taken and adapter from here: https://github.com/onflow/flow-go/blob/master/integration/benchmark/account/keys.go#L20-L110